src/js/constants/ServiceStatusTypes.js

```diff
@@ -1,7 +1,8 @@
 var ServiceStatusTypes = {
   RUNNING: 0,
   DEPLOYING: 1,
-  SUSPENDED: 2
+  SUSPENDED: 2,
+  NA: 4
```

Comment:
In marathon-ui the `4` is `waiting`. Maybe we should have this in mind, not sure. Feel free to ignore.